### PR TITLE
docs: Add production installation ECS docs

### PR DESCRIPTION
**Pull request by ishustava**

### CODE REVIEW

File: website/content/docs/ecs/get-started/production-installation.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+update in AWS Secrets Manager.
+
+-> **NOTE:** Make sure that the `name_prefix` is unique for each ECS cluster where you are
+deploying this controller.
+
+## Deploy Services
+
+Once ACL controller is up and running, you can deploy services on the mesh using the [`mesh-task` module](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task).
+Please make sure that you already have the basic configuration for the [Task Module](/docs/ecs/get-started/install#task-module).
+We will add additional configuration to that basic configuration to make it production-ready.
```
Comment (trujillo-adam) [inline website/content/docs/ecs/get-started/production-installation.mdx:72]:
```suggestion
Start with the basic configuration for the [Task Module](/docs/ecs/get-started/install#task-module) and specify additional settings  to make the configuration production-ready.
```
For consistent style

**Review by trujillo-adam — APPROVED** [review]
Just added a few suggestions to make the writing more consistent with the style guide.

**Review by lkysow — APPROVED** [review]
This looks great

File: website/content/docs/ecs/get-started/production-installation.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+layout: docs
+page_title: Production Installation - AWS ECS
+description: >-
+  Production Installation of the Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Production Installation
+
+For a production-ready installation of Consul on ECS, you will need to make sure that the cluster is secured.
+Secure Consul cluster should include the following:
```
Comment (pglass) [inline website/content/docs/ecs/get-started/production-installation.mdx:11]:
```suggestion
A secure Consul cluster should include the following:
```

File: website/content/docs/ecs/get-started/production-installation.mdx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+and CA certificate for the Consul server in AWS Secrets Manager.
+
+```hcl
+resource "aws_secretsmanager_secret" "bootstrap_token" {
+  name  = "bootstrap-token"
+}
+
+resource "aws_secretsmanager_secret_version" "bootstrap_token" {
+  secret_id     = aws_secretsmanager_secret.bootstrap_token[count.index].id
+  secret_string = random_uuid.bootstrap_token[count.index].result
```
Comment (pglass) [inline website/content/docs/ecs/get-started/production-installation.mdx:34]: I think we can remove `count.index` in this example.

**Review by pglass — APPROVED** [review]
Looks great! 👍

File: website/content/docs/ecs/get-started/production-installation.mdx (hunk 3 above)
Comment (ishustava) [reply website/content/docs/ecs/get-started/production-installation.mdx:34]: lol thank you! yes this was a bad copy-pasta 😄

Comment (ishustava) [inline website/content/docs/ecs/get-started/production-installation.mdx:34]:
```suggestion
  secret_id     = aws_secretsmanager_secret.bootstrap_token.id
  secret_string = random_uuid.bootstrap_token.result
```

**Pull request merged**